### ISSUE TITLE
`add_process` most basic example doesn't work

### ISSUE THREAD

**Issue by alahiff** [simvue-io/python-api#439]
Following https://docs.simvue.io/tracking-monitoring/launching_processes/ using the client from PyPI:
```
run.add_process("my_process", "echo", "Hello World!")
```
gives:
```
FileNotFoundError: [Errno 2] No such file or directory: 'None'
```
For some reason `add_process` always replaces the executable with `None`, at least it does for me.

**Comment by alahiff** [comment]
It seems like you have to do the following for it to work:
```
run.add_process("my_process", "Hello World!", executable="echo")
````

**Comment by kzscisoft** [comment]
Addressed in `v1.0.1-rc` branch
